parser-typechecker/src/Unison/Parser.hs

```diff
@@ -244,8 +244,7 @@ run' p s name env =
             then L.lexer name (trace (L.debugLex''' "lexer receives" s) s)
             else L.lexer name s
       pTraced = traceRemainingTokens "parser receives" *> p
-      env' = env { names = Names.suffixify (names env) }
-  in runParserT pTraced name (Input lex) env'
+  in runParserT pTraced name (Input lex) env
```

Comment:
Parser no longer needs a suffixed names since the names index can directly resolve unique suffixes.